### ISSUE TITLE
News collected in category groups

### ISSUE THREAD

**Issue by jankout** [codefog/contao-news_categories#152]
It would be nice to have a module to make a news list collected in a chosen categories with a category title (optional). 

_News category name
- related news

News category name
- related news

etc._

**Comment by asaage** [comment]
That would result in a news-item apearing multiple times in the list if it has multiple categories wouldn't it?

**Comment by jankout** [comment]
Yes, nur there could be a possibility to choose whether it should be repeated or not.

**Comment by qzminski** [comment]
Sounds like a good idea. Pull requests are welcome 😉
